tests/gui/base/test_inputs.py

```diff
@@ -0,0 +1,14 @@
+from PyQt5.QtWidgets import QApplication, QLineEdit
+
+from securedrop_client.gui.base import PasswordEdit
+
+app = QApplication([])
+
+
+def test_PasswordEdit(mocker):
+    passwordline = PasswordEdit(None)
+    passwordline.togglepasswordAction.trigger()
+
+    assert passwordline.echoMode() == QLineEdit.Normal
+    passwordline.togglepasswordAction.trigger()
+    assert passwordline.echoMode() == QLineEdit.Password
```

Comment:
Test is ported to its new location, but not removed from its old. Fix and squash in?